### DISCUSSION TITLE
How to not load intermediate model data in deep relationship?

### DISCUSSION THREAD

**Discussion by yogeshgalav** [laravel/framework#48940] [Q&A] [answered]
I have customers table containing business_id
and care_managers table containing business_id as well

now in my controller code I was loading all care_managers available for customer
```
$customer->load([
    'business.careManagers:business_id,name,email',
]);
```
Is it possible to **not** show or send business data?

**Comment by chuckrincon** [comment, marked as answer]
You could use the [Has One Through](https://laravel.com/docs/10.x/eloquent-relationships#has-one-through) or [Has Many Through](https://laravel.com/docs/10.x/eloquent-relationships#has-many-through) to achieve this.
